src/main/java/com/box/sdk/BoxFolder.java

```diff
@@ -876,6 +878,77 @@ public void deleteMetadata(String templateName, String scope) {
     }
 
+    /**
+     * Adds a metadata classification to the specified file.
+     *
+     * @param classificationType the metadata classification type.
+     * @return the metadata classification type added to the file.
+     */
+    public String addClassification(String classificationType) {
+        Metadata metadata = new Metadata().add("/Box__Security__Classification__Key", classificationType);
+        Metadata classification = this.createMetadata(CLASSIFICATION_TEMPLATE_KEY, "enterprise", metadata);
+
+        return classification.getString("/Box__Security__Classification__Key");
+    }
+
+    /**
+     * Updates a metadata classification on the specified file.
+     *
+     * @param classificationType the metadata classification type.
+     * @return the new metadata classification type updated on the file.
+     */
+    public String updateClassification(String classificationType) {
+        Metadata metadata = this.getMetadata(CLASSIFICATION_TEMPLATE_KEY);
+        metadata.replace("/Box__Security__Classification__Key", classificationType);
+        Metadata classification = this.updateMetadata(metadata);
+
+        return classification.getString("/Box__Security__Classification__Key");
+    }
+
+    /**
+     * Attempts to add classification to a file. If classification already exists then do update.
+     *
+     * @param classificationType the metadata classification type.
+     * @return the metadata classification type on the file.
+     */
+    public String setClassification(String classificationType) {
+        Metadata metadata = new Metadata().add("/Box__Security__Classification__Key", classificationType);
+        Metadata classification = null;
+
+        try {
+            classification = this.createMetadata(CLASSIFICATION_TEMPLATE_KEY, "enterprise", metadata);
+        } catch (BoxAPIException e) {
+            if (e.getResponseCode() == 409) {
+                metadata = this.getMetadata(CLASSIFICATION_TEMPLATE_KEY);
+                metadata.replace("/Box__Security__Classification__Key", classificationType);
+                classification = this.updateMetadata(metadata);
+            } else {
+                throw e;
+            }
+        }
+
+        return classification.getString("/Box__Security__Classification__Key");
+    }
+
+    /**
+     * Gets the classification type for the specified file.
+     *
+     * @return the metadata classification type on the file.
+     */
+    public String getClassification() {
+        Metadata metadata = this.getMetadata(CLASSIFICATION_TEMPLATE_KEY);
+        return metadata.getString("/Box__Security__Classification__Key");
+    }
+
+    /**
+     * Deletes the classification on the file.
+     */
+    public void deleteClassification() {
+        Metadata metadata = this.getMetadata(CLASSIFICATION_TEMPLATE_KEY);
+        metadata.remove("/Box__Security__Classification__Key");
+        Metadata classification = this.updateMetadata(metadata);
```

Comment:
See note in `BoxFile.java` — this should probably remove the template altogether